gridftp/control/source/test/connect_test.c

```diff
@@ -1135,7 +1129,6 @@ initialize_servers(void)
         globus_io_tcpattr_destroy(&tls_server_attr);
     }
 failed_init_cleartext:
-failed_other_attr_init:
```

Comment:
An unused label gives a compiler warning. I guess it could be kept commented out. But that only makes sense if there is some commented out code that has a goto to the label, I think. Otherwise it is just confusing.

Reply:
I agree, without any other reference to the label, it will only create confusion.